week2.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>What is library in R?</p><p>What is tidyverse in R?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>